gadopt/approximations.py

```diff
@@ -432,49 +518,288 @@ class SmallDisplacementViscoelasticApproximation:
 
     N.b. that the implentation currently assumes all terms are dimensional.
```

Comment:
`implementation`

Is it still true that all terms are dimensional only?

Reply:
Yeah sorry I never ended up incorporating your nondimensional changes to the old way of zhong doing things